src/core/server/ui_settings/settings/theme.ts

```diff
@@ -27,7 +27,7 @@ export const getThemeSettings = (): Record<string, UiSettingsParams> => {
       name: i18n.translate('core.ui_settings.params.themeVersionTitle', {
         defaultMessage: 'Theme version',
       }),
-      value: 'v7',
+      value: 'v8 (beta)',
       type: 'select',
       options: ['v7', 'v8 (beta)'],
```

Comment:
Should we remove this `(beta)` now that we're going to use it as default?

Reply:
++ Makes sense to me

Reply:
For people who already have `v8 (beta)` in their advanced settings wouldn't this break things? Do we validate the values in ES and revert to the default when the saved value is not in the options?

Reply:
Actually, looked at the places we're consuming the `theme:version` setting we are only checking for `v7` so changing to `v8` should be fine.

Reply:
> Do we validate the values in ES and revert to the default when the saved value is not in the options

No, we don't. We're just logging a pretty much useless warning in that case...

We could change the label without changing the value using the `optionLabels` property of the `UiSettingsParams` though.

The correct solution would probably be to add a migration to `src/core/server/ui_settings/saved_objects/migrations.ts` for that.